include/irr/builtin/material_compiler/glsl/common.glsl

```diff
@@ -700,44 +578,30 @@ void flipCurrInteraction()
 	currInteraction.BdotV = -currInteraction.BdotV;
 }
 //call before executing an instruction/evaluating bsdf
-void handleTwosided_interactionOnly(inout bool ts_flag, in instr_t instr)
+void handleTwosided_interactionOnly(in instr_t instr)
```

Comment:
why do you have like 4 functions for the same thing?

Reply:
? what 4 functions?